Lib/idlelib/configdialog.py

```diff
@@ -735,16 +737,16 @@ def create_page_general(self):
         win_height_title = Label(frame_win_size, text='Height')
         self.entry_win_height = Entry(
                 frame_win_size, textvariable=self.win_height, width=3)
```

Comment:
With the issue mentioned on the ticket, would this help for validating the entry widget?
https://stackoverflow.com/questions/4140437/interactively-validating-entry-widget-content-in-tkinter#4140988